config/config_test.go

```diff
@@ -1379,53 +1261,6 @@ func TestValidateAccountsConfigRestrictions(t *testing.T) {
 	assert.Contains(t, errs, errors.New("accounts.postgres: retrieving accounts via postgres not available, use accounts.files"))
 }
 
-func TestUserSyncFromEnv(t *testing.T) {
```

Comment:
Why is this test being removed? Is it intended that it should no longer be possible to set usersync parameters from environment variables?

Reply:
There will be no adapter related config in environment variables. All bidder related data is loading from {bidder}.yaml files or pbs.json or pbs.yaml files. 
`cfg.Adapters` were removed from Configuration struct. 

`assert.Nil(t, cfg.Adapters["appnexus"].Syncer.IFrame)` - is not a valid structure anymore.

Reply:
It is common for kubernettes deployments to use environment variables for configuration rather than config files, particularly when you have deployments in multiple datacenters that have datacenter specific parameters (such as the bidder endpoint). Is this breaking change already documented in an issue? Otherwise we should certainly document it as such and give hosts ample opportunity to rework their setups to continue to work properly when this feature is removed.

Reply:
Did you add back in tests for environment variables setting adapter parameters somewhere else?

Reply:
Good point! Added and fixed mapstructures in bidder info and added remaining bidder info properties to `setBidderDefaults`